src/components/Home.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import getData from '../actions/actionCreator';
+import microphone from '../images/microphone.png';
+import leftArrow from '../images/left-arrow.png';
+import right from '../images/right.png';
+import settings from '../images/settings.png';
+
+const Home = () => {
+  const covidData = useSelector((data) => data.Reducer);
+  const dispatch = useDispatch();
+  const date = '2020-10-10';
+  const APIURL = `https://api.covid19tracking.narrativa.com/api/${date}`;
+  useEffect(() => {
+    const covidApi = async () => {
+      const fetchAPI = await fetch(APIURL);
+      const covidData = await fetchAPI.json();
+      const data = covidData.dates[date].countries;
+      const newData = Object.values(data);
+      return dispatch(getData(newData));
+    };
+    covidApi();
+  }, []);
```

Comment:
- [ ] Nice display of all the countries on your Home page. But as the previous Code Reviewer suggested, you can add a filter stateless component to your App as this is a requirement of this project. The filter component can filter the countries for example if I want to search for a country, I can just go to the filter bar and type in this country and it will be filtered from all the other countries and then shown to me